source/includes/_snapshots.md

```diff
@@ -1 +1,126 @@
 # Snapshots
+
+## The Snapshot Resource
+
+### Attributes:
+
+
+Attribute | Type | Description
+--------- | ------- | -------
+`id` | string | A unique identifier of this snapshot.
+`created_at` | datetime | The time the snapshot was uploaded to or created on the manager.
```

Comment:
Please change datetime to ISO 8601 as it appears here: http://docs.getcloudify.org/api/#events